game/game_application.cpp

```diff
@@ -14,6 +14,21 @@
 
 #include "systems/game_render_system.h"
 
+bool checkReservedName(std::string sceneName)
+{
+	std::vector<std::string> reserved_names_list { "pause.scene", "pauseUI.scene" };
+	for (auto& reservedName : reserved_names_list)
```

Comment:
Follow the naming convention
Rename it to `reservedNames`
use `Vector`
https://github.com/sdslabs/Rootex/blob/e4cc841fb5bfb7ca848e671b005f3e92e3b1391d/rootex/common/types.h#L99

Reply:
done